padl/transforms.py

```diff
@@ -1743,10 +1861,56 @@ def __call__(self, args):
         :param args: Arguments to call with.
         :return: Output from series of transforms.
         """
-        for transform_ in self.transforms:
-            args = transform_.pd_call_transform(args)
+        _in_args = args
+        for i, transform_ in enumerate(self.transforms):
+            try:
+                args = transform_.pd_call_transform(args)
+            except Exception as err:
+                self._pd_trace_error(i, _in_args)
```

Comment:
@eramdiaz What are the `_in_args` used for in the traceback?

Reply:
I wanted to make sure that the `args` we are keeping on the trace are the first ones that are input to the `Compose` and not intermediate ones, but it might be not needed

Reply:
I see. I'm not sure I'm a huge fan of this because it could potentially cause a lot of overhead since we are keeping a copy of the input args. For example, if the input args happen to be a large matrix we are doubling the memory overhead.

Reply:
I'll keep this in mind and try to find an alternative

Reply:
Would it be possible to save `in_args` in disk with the `padl` value save in a temporary file?

Reply:
It would be possible, but you shouldn't do that as it would potentially create a lot of new problems (you would need to keep track of the files, find a good way to serialize any kind of arg etc).

As mentioned before, I don't think we have a problem here because nothing is being copied.

Reply:
You're right. It would only delay in garbage collecting of `_in_args`.